source/quickstart.rst

```diff
@@ -9,22 +9,32 @@ Welcome to VAlgoLang's documentation!
 VAlgoLang is a domain specific language that helps educators create animations for teaching students about data structures
 and algorithms.
 
-Here is an example:
+Here's a short example:
 
 .. code:: javascript
 
-    let stack: Stack = new Stack;
+    let stack = Stack<number>();
     stack.push(5);
-    stack.pop();
+    let x: number = stack.pop();
 
 This would produce the video:
 
 .. raw:: html
 
         <video src="_static/intro.mp4" frameborder="0" allowfullscreen style=" width: 80%; height: 80%;" controls></video>
```

Comment:
we probably need to change the video too right?

Reply:
I changed it! Just gave it the same name :)